package.json

```diff
@@ -14,22 +14,24 @@
     "build": "npm run build:lib && npm run build:umd && npm run build:umd:min && npm run dts",
     "build:lib": "tsc && babel lib --out-dir lib",
     "build:umd": "webpack src/index.ts dist/mobx-forms.js",
-    "build:umd:min": "NODE_ENV=production webpack src/index.ts dist/mobx-forms.min.js",
+    "build:umd:min": "set NODE_ENV=production&&webpack src/index.ts dist/mobx-forms.min.js",
```

Comment:
this will break things for non Windows users though. you can use `crossenv` to fix this issue